### DISCUSSION TITLE
Is it possible to add "--prune" in pull?

### DISCUSSION THREAD

**Discussion by shino** [magit/magit#4813] [Support] [answered]
First of all, thank you very much for the excellent software!

I usually use "--prune" in fetch, as "f -p" in Magit way (also, it's saved as default).

Today, I noticed that "git pull" also support "--prune" option like:

```
% git pull --all --prune
Already up to date.
```

In Magit's pull menu, "--prune" is not listed. Is it possible to use the option in magit?

**Comment by lafrenierejm** [comment, marked as answer]
See the variable `magit-pull-or-fetch`, which was added in magit 3.0.0.

> Whether magit-pull also offers some fetch suffixes.

Setting that varialbe to `t` expose a "pull and prune" command in pull's transient.

**Comment by shino** [comment]
Thanks!
